presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -710,6 +710,25 @@ public void testDropTableIfExists()
         assertFalse(getQueryRunner().tableExists(getSession(), "test_drop_if_exists"));
     }
 
+    @Test
+    public void testMaterializedView()
```

Comment:
We should validate:
- `SHOW CREATE TABLE` of the materialized view will fail.
- `SHOW CREATE VIEW` of the materialized view will fail.
- Any DML (insert, delete, update, etc) on the materialized view will fail.

Reply:
- MV on top of an MV
- partition mapping invalid vs valid
- more MV definition including agg on top of agg, join, union all, window
- add a todo: "deleting the base table will automatically drop all the MVs"
- retention related tests

Reply:
Retention is validated in the presto-facebook repo (retention_days is a fb property). Do you happen to know the Test class in facebook repo? Or I will skip.